examples/rosenbrock_2d_delayed.py

```diff
@@ -1,5 +1,19 @@
 import time
 
+# --------------------------------------------------------------
+# When working with multiprocessing, we need to provide a pickable
+# function and use __main__. Details can be found on:
+# https://docs.python.org/3.2/library/multiprocessing.html#multiprocessing-programming
```

Comment:
Please like to: https://docs.python.org/3/library/multiprocessing.html#multiprocessing-programming

Reply:
Done